src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidDataException;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "normal";
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final int DEFAULT_AGE = 18;
+    private static final String TEST_INVALID_LOGIN = "";
+    private static final String TEST_INVALID_PASSWORD = "123";
+    private static final int TEST_INVALID_AGE = 10;
+    private static final int TEST_NEGATIVE_AGE = -5;
```

Comment:
In your test classes that is OK if you use literals directly instead of constants. But, if you already have created a lot of constants, you can move them to a separate class, e.g. RegistrationServiceTestUtils or RegistrationServiceTestConstants